lib/evva/analytics_event.rb

```diff
@@ -1,13 +1,17 @@
 module Evva
   class AnalyticsEvent
-    attr_reader :event_name, :properties
-    def initialize(event_name, properties = {})
+    attr_reader :event_name, :properties, :platforms
```

Comment:
We probably should call this `destinations` instead of `platforms` to avoid confusion with Android/iOS

```suggestion
    attr_reader :event_name, :properties, :destinations
```